internal/services/datadog/azurerm_datadog_monitor_sso_configuration_test.go

```diff
@@ -64,7 +64,6 @@ func TestAccDatadogMonitorSSO_requiresImport(t *testing.T) {
 			Config: r.basic(data),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
-				check.That(data.ResourceName).Key("single_sign_on_enabled").HasValue("Enable"),
```

Comment:
We'll want to update the tests to use the new value and keep a legacy test config around

Reply:
Roger. I added a test `TestAccDatadogMonitorSSO_singleSignOnEnabled` for the legacy attribute